src/util.py

```diff
@@ -260,12 +265,156 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def findLocalExtrema(da, highVal=1040, lowVal=975, eType='Low'):
+    """
+    Utility function to find local low/high field variable coordinates on a contour map. To classify as a local high, the data
+    point must be greater than highVal, and to classify as a local low, the data point must be less than lowVal.
+
+    Args:
+
+        da: (:class:`xarray.DataArray`):
+            Xarray data array containing the lat, lon, and field variable (ex. pressure) data values
+
+        highVal (:class:`int`):
+            Data value that the local high must be greater than to qualify as a "local high" location.
+            Default highVal is 1040.
```

Comment:
Don't forget to mention that the defaults are assuming that the data is pressure in hectopascals (hPa)